### PR TITLE
Conic projection descriptions

### PR DESCRIPTION
**Pull request by willschlitzer**
Adding projection descriptions for the conic gallery examples, as discussed in #747.

These projection descriptions are copied directly from the GMT documentation.

### CODE REVIEW

File: examples/projections/conic/conic_albers.py [hunk 1]
```diff
@@ -1,9 +1,20 @@
 """
 Albers Conic Equal Area
 =======================
+This projection, developed by Heinrich C. Albers in 1805, is predominantly used to map
```
Comment (seisman) [inline examples/projections/conic/conic_albers.py:4]: I think the source codes look better if we add a blank space between the section title and the paragraph, but it's just my personal taste.

```suggestion
=======================

This projection, developed by Heinrich C. Albers in 1805, is predominantly used to map
```

Comment (willschlitzer) [reply examples/projections/conic/conic_albers.py]: Just pushed a change with a blank line added to all of the docstrings

File: examples/projections/conic/conic_albers.py [hunk 2]
```diff
@@ -2,8 +2,20 @@
...
+other hand, are equally spaced radii about a common center, and cut the parallels at
+right angles. Distortion in scale and shape vanishes along the two standard parallels.
+Between them, the scale along parallels is too small; beyond them it is too large.
+The opposite is true for the scale along meridians.
+
 ``Blon0/lat0/lat1/lat2/width``: Give projection center ``lon0/lat0`` and two standard
 parallels ``lat1/lat2``.
+
+For further information, see
+:gmt-docs:`cookbook/map-projections.html#albers-conic-equal-area-projection-jb-jb`
```
Comment (seisman) [inline examples/projections/conic/conic_albers.py:18]: I think we can remove these long links to the GMT cookbook.

Comment (willschlitzer) [reply examples/projections/conic/conic_albers.py]: Removed.

**Review by seisman — APPROVED** [review]

**Pull request merged**